hal/src/main/native/include/hal/CANAPITypes.h

```diff
@@ -56,7 +56,9 @@ HAL_ENUM(HAL_CANManufacturer) {
   HAL_CAN_Man_kPWF = 11,
   HAL_CAN_Man_kStudica = 12,
   HAL_CAN_Man_kTheThriftyBot = 13,
-  HAL_CAN_Man_kReduxRobotics = 14
+  HAL_CAN_Man_kReduxRobotics = 14,
+  HAL_CAN_MAN_kAndyMark = 15,
```

Comment:
Is there a reason why this one is `MAN` instead of `Man`?

Reply:
I think a slow finger letting go of the shift key would be the reason